config/quota-management-list-configuration.yaml

```diff
@@ -5,25 +5,62 @@
 # In the future it may support service accounts outside RH.
 # The structure of registered service accounts is:
 #       - username: is the account of the user. The username must be unique
-#       - max_allowed_instances: is the maximum number of instances this user can create.
+#       - max_allowed_instances: is the maximum number of instances of a given instance_type/billing_model pair this user can create.
+#         This value is used if `allowed` field (inside `billing_model`) is 0 or omitted.
 #         Defaults to the global value of `max-allowed-instances` which has different values for distinct environments.
+#       - granted_quota: the configuration of the quota granted to the user. The configuration is defined as follows:
+#         - instance_type_id: the id of the instance type for which we are configuring the quota
+#         - billing_models: the list of billing models that we are going to configure. This is defined as follows:
```

Comment:
when we talk about billing_models here what are we talking about? is it kafka billing models? ams billing models? another billing model concept independent of those?

Reply:
This is the quota_list, ams is not related to this.
As in AMS we have a billin_model in the AMS configuration (saying how much of a resource with a specified billing model is allowed), here we have a billing_model in the quota_list saying how many `instance_type` instances are allowed for a specific billing_model.
The set of accepted billing models here is the list of configured kafka_billing_models for the given instance type.

Reply:
Could we call them then kafka_billing_models instead?